.github/CONTRIBUTING.md

```diff
@@ -113,12 +113,17 @@ $ git push -u origin
 
 and publish the PR in GitHub's web interface!
 
+After your pull request is merged and the branch is no longer needed, delete it:
+
+```
+$ git checkout main
+$ git push --delete origin my_topical_branch && git branch -d my_topical_branch
+```
+
 Before starting to work on your next pull request, run the following command to sync your local repository with the remotes:
 
 ```console
 $ git fetch --all
-$ git checkout main
-$ git merge
```

Comment:
Why remove this? Doesn’t that make the local main branch permanent out-of-date? I imagine that being very confusing.

Reply:
See the [comment](https://github.com/python-attrs/attrs/pull/1105#discussion_r1156118863) by @webknjaz

Reply:
I think we're the breaking reasonable assumption that `git checkout main` gives you something useful, no?

I might be just too dim, but it doesn't sound to me like the comments by @webknjaz apply to someone who needs a Git 101 tutorial. I feel like we're getting a little bit lost in the weeds and end up with a book on Git. 😅 I'm happy to be convinced otherwise of course – I'm a bit over my head herre since I don't use Git like any of this _at all_. (I use mostly GitHub's features like "Sync Fork" or `gh pr checkout` etc).

Reply:
(I'm not trying to be difficult, I'm just trying to understand what's in this file that's supposed to be helpful 😅)

Reply:
How about `git fetch --all && git merge main`?

Reply:
Does that work if one is not on main?